app/models/miq_group.rb

```diff
@@ -109,6 +116,16 @@ def self.seed
         seq += 1
       end
     end
+
+    # find any default tenant groups that do not have a role
+    tenant_role = MiqUserRole.default_tenant_role
+    if tenant_role
+      tenant_groups.where(:miq_user_role_id => nil).each do |group|
+        group.update_attributes(:miq_user_role => tenant_role)
+      end
+    else
+      _log.warn("Unable to find default tenant role for tenant access")
+    end
```

Comment:
This feels migrationy, and should not be in a seed.  The reason being that we will have to carry this code forever as a "just in case", whereas in a migration it's one and done.